pkg/cmd/workflow/shared/shared.go

```diff
@@ -182,7 +177,7 @@ func getWorkflowsByName(client *api.Client, repo ghrepo.Interface, name string,
 	return filtered, nil
 }
 
-func ResolveWorkflow(io *iostreams.IOStreams, client *api.Client, repo ghrepo.Interface, prompt bool, workflowSelector string, states []WorkflowState) (*Workflow, error) {
+func ResolveWorkflow(p iprompter, io *iostreams.IOStreams, client *api.Client, repo ghrepo.Interface, prompt bool, workflowSelector string, states []WorkflowState) (*Workflow, error) {
```

Comment:
Would be nice to simplify/reduce the number of arguments to this function as they are getting a bit unwieldily but that is outside the scope of this work.